libs/sdk-bindings/bindings-swift/Sources/BreezSDKNotification/Task/LnurlPayInfo.swift

```diff
@@ -65,10 +65,13 @@ class LnurlPayInfoTask : LnurlPayTask {
             }
             replyServer(encodable: LnurlInfoResponse(callback: request!.callback_url, maxSendable: maxSendable, minSendable: minSendable, metadata: metadata, tag: "payRequest"),
                         replyURL: request!.reply_url)
-        } catch let e as InvalidMinSendable.largerThanMaxSendable {
-            self.logger.log(tag: TAG, line: "failed to process lnurl: \(e)", level: "ERROR")
-            let failNotificationTitle = ResourceHelper.shared.getString(key: Constants.LNURL_PAY_NOTIFICATION_LIQUIDITY_FAILURE_TITLE, fallback: Constants.DEFAULT_LNURL_PAY_NOTIFICATION_LIQUIDITY_FAILURE_TITLE)
-            fail(withError: e.localizedDescription, replyURL: request!.reply_url, failNotificationTitle: failNotificationTitle)
+        } catch let e as InvalidMinSendable {
+            switch e {
+            case .largerThanMaxSendable:
+                self.logger.log(tag: TAG, line: "failed to process lnurl: \(e)", level: "ERROR")
+                let failNotificationTitle = ResourceHelper.shared.getString(key: Constants.LNURL_PAY_NOTIFICATION_LIQUIDITY_FAILURE_TITLE, fallback: Constants.DEFAULT_LNURL_PAY_NOTIFICATION_LIQUIDITY_FAILURE_TITLE)
+                fail(withError: e.localizedDescription, replyURL: request!.reply_url, failNotificationTitle: failNotificationTitle)
+            }
```

Comment:
Since `InvalidMinSendable` only has `largerThanMaxSendable` case, `default` case is obsolete and won't be executed.